odl/space/ntuples.py

```diff
@@ -1320,7 +1349,7 @@ def adjoint(self):
                                       ''.format(self.domain.field,
                                                 self.range.field))
         return MatVecOperator(self.range, self.domain,
-                              np.asarray(np.asmatrix(self.matrix).H))
+                              self.matrix.conj().T)
```

Comment:
Yeah i mean that it would be a `numpy.matrix`, maybe some name of what it does instead of what it is?